### DISCUSSION TITLE
German translater

### DISCUSSION THREAD

**Discussion by GameTec-live** [borgbase/vorta#1042] [Translations]
Ive seen a massage from the founder on reddit, that translaters are welcome and they just have to leave a issue. So... i know a much bit of python and i know german and english.

**Comment by m3nu** [comment]
Always! Have you already found our translation docs? Just follow the link to Transifex there and you will have full access to translations.

**Comment by m3nu** [comment]
(moved to Translation discussions)

**Comment by max-foss** [comment]
I am currently working on the German translation and would like to push it over the 100% line and give the existing translations some more polish.

I would like to move to either consistent **du** (my preference) or **sie** instead of the current mix, which feels quite odd to me. Any other German translators want to chime in?

Also, it seems like new strings need to be pushed to Transifex by someone with the appropriate permissions.

**Reply by m3nu** [reply]
Go with 'du'.

My opinion is that whoever does the work decides.

I think you can take over the German translation. Initial version was done by Thomas, but I assume he's more focused on Borg right now.

**Reply by m3nu** [reply]
I'll do a beta release when you tell me translations are done. We're overdue anyways.

**Reply by max-foss** [reply]
It'd be very helpful to have at least reviewer status for the German translation that I can mark translations as reviewed to avoid having to keep track of them elsewhere.

Also, I have a hunch there are a few missing strings and it says **Source Updated Oct 18th 2024, 14:59**. Could you push an update of the strings?

Please ignore me joining the Dutch language, too. I am not that advanced in Dutch and won't contribute there but I very much appreciate that quality of the translation.

**Reply by m3nu** [reply]
Both fine with me. Made you a reviewer and pulled+pushed translations.

**Reply by max-foss** [reply]
Hmm, for example the untranslated string **Try to replace file permissions when mounting an archive** still does not show up in Transifex for me. Any idea what might cause this?

**Reply by m3nu** [reply]
On checking the source file, it should show up since it's marked for translation correctly. I need to look into it more.
